src/lib/util.ts

```diff
@@ -301,7 +301,7 @@ export async function getNetworkId(web3Provider?: Web3Provider): Promise<string>
       return network.chainId.toString();
     } else {
       const promise = new Promise<string>((resolve, reject) => {
-        web3Provider.send("net_version", (error, response) => {
+        web3Provider.send("net_version", (error: any, response: any) => {
```

Comment:
Is it related?

Reply:
it is just the usual typescript lint warning improvements, an ongoing improvement...does not affect execution of the code.

Reply:
ok, yes I assume also the `boolean` in `proposalHelpers.ts`